src/main/kotlin/com/exactpro/th2/check1/CollectorService.kt

```diff
@@ -187,7 +189,8 @@ class CollectorService(
                 val messageID = sessionKey.toMessageID(sequence)
                 rootEvent.messageID(messageID)
                     .addSubEventWithSamePeriod()
-                    .name("Checkpoint for session alias '${sessionKey.sessionAlias}' direction '${sessionKey.direction}' sequence '$sequence'")
+                    .bookName(request.parentEventId.bookName)
```

Comment:
Let's pass a book name and scope from a parent Event in the `addSubEventWithSamePeriod` method and others that build sub-events.

Reply:
There are 2 methods for sub events:
```
public Event addSubEventWithSamePeriod() {
    return addSubEvent(new Event(startTimestamp, endTimestamp));
}
public Event addSubEvent(Event subEvent) {
    subEvents.add(requireNonNull(subEvent, "Sub event can't be null"));
    return subEvent;
}
```
Should I add `bookName` and `scope` from parent to both of them?
`addSubEvent()` accepts `Event`, what if it already has bookName or scope or it it incorrect situation?

Reply:
As discussed on meeting I'll get `bookName` and `scope` from `parentID` in `Event.toProto()`.
Also I'll add another method to `common.Event` like `toProto()` which will accept `bookName` and `scope`.